### PR TITLE
[DOCS] Reformats nodes feature usage API

### PR DESCRIPTION
**Pull request by szabosteve**
Relates to elastic/docs#937 and https://github.com/elastic/elasticsearch/issues/45197.

This PR updates the nodes feature usage API to align with the new [API reference template](https://github.com/elastic/docs/blob/master/shared/api-ref-ex.asciidoc).

Resources:
* [Cluster nodes feature usage API spec](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/nodes.usage.json)
* [Cluster nodes feature usage API doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-usage.html)

### CODE REVIEW

**Comment by szabosteve** [comment]
@elasticmachine test this please

File: docs/reference/cluster/nodes-usage.asciidoc [hunk 1]
```diff
@@ -1,33 +1,62 @@
...
-GET _nodes/usage
-GET _nodes/nodeId1,nodeId2/usage
---------------------------------------------------
-// CONSOLE
-// TEST[setup:node]
-// TEST[s/nodeId1,nodeId2/*/]
+[[cluster-nodes-usage-api-request]]
+==== {api-request-title}
+
+`GET _nodes/usage` +
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-usage.asciidoc]: I'd add a leading slash for consistency.
```suggestion
`GET /_nodes/usage` +
```

File: docs/reference/cluster/nodes-usage.asciidoc [hunk 2]
```diff
@@ -1,33 +1,62 @@
...
+
+`GET _nodes/{node_id}/usage` +
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-usage.asciidoc]:
```suggestion
`GET /_nodes/{node_id}/usage` +
```

File: docs/reference/cluster/nodes-usage.asciidoc [hunk 3]
```diff
@@ -1,33 +1,62 @@
...
 
-The first command retrieves usage of all the nodes in the cluster. The
-second command selectively retrieves nodes usage of only `nodeId1` and
-`nodeId2`. All the nodes selective options are explained
+`GET _nodes/usage/{metric}` +
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-usage.asciidoc]:
```suggestion
`GET /_nodes/usage/{metric}` +
```

File: docs/reference/cluster/nodes-usage.asciidoc [hunk 4]
```diff
@@ -1,33 +1,62 @@
...
+
+`GET _nodes/{node_id}/usage/{metric}`
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-usage.asciidoc]:
```suggestion
`GET /_nodes/{node_id}/usage/{metric}`
```

File: docs/reference/cluster/nodes-usage.asciidoc [hunk 5]
```diff
@@ -1,33 +1,62 @@
...
+
+
+[[cluster-nodes-usage-api-desc]]
+==== {api-description-title}
+
+The cluster nodes usage API allows to retrieve information on the usage
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-usage.asciidoc]:
```suggestion
The cluster nodes usage API allows you to retrieve information on the usage
```

**Review by jrodewig — APPROVED** [review]
Some minor formatting comments. Otherwise LGTM.

File: docs/reference/cluster/nodes-usage.asciidoc [hunk 6]
```diff
@@ -1,33 +1,62 @@
...
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+
+[[cluster-nodes-usage-api-example]]
+==== {api-examples-title}
+
+Rest action example:
+
+`GET _nodes/usage/rest_actions`
```
Comment (jrodewig) [inline docs/reference/cluster/nodes-usage.asciidoc]: Sorry I missed this earlier. You'll want to change this example to the following. Should fix the gradle issues you're seeing.
```suggestion
[source,js]
--------------------------------------------------
GET _nodes/usage
--------------------------------------------------
// CONSOLE
// TEST[setup:node]
```

**Comment by szabosteve** [comment]
Backported to 7.x and 7.3.

**Pull request merged**